pyproject.toml

```
@@ -128,8 +128,8 @@ gtk-platform = [
 
 mypy = [
     "twisted[dev,all-non-platform]",
-    "mypy~=1.8",
-    "mypy-zope~=1.0.3",
+    "mypy<1.11",
+    "mypy-zope",
```

My understanding is that the `twisted[mypy]` deps are only to be install for development work.

They don't need to be installed for "production"

So `twisted[mypy]` is kind of a private extra deps ...

This mean that we should be able to pin the exact version... nobody else should depend on `twisted[mypy]`

```suggestion
    "mypy==1.11",
    "mypy-zope==1.0.6",
```

---

Can't quite apply the suggestion because 1.11 is what broke it, so upgrading will be a separate issue.